src/Middleware/HostFiltering/test/HostFilteringMiddlewareTests.cs

```diff
@@ -97,6 +97,37 @@ public async Task AllowsEmptyHost(bool allowed, int status)
             Assert.Equal(status, (int)response.StatusCode);
         }
 
+        [Fact]
+        public async Task WhitespaceHostRejected()
```

Comment:
@Tratcher I okay if we take this change (to reject whitespace hosts). Thoughts?

Reply:
Dropped this rejection https://github.com/aspnet/AspNetCore/pull/9359#discussion_r282624054 as it should never happen; so should drop the test too

Reply:
As the whitespace reject has gone; this test is just checking if whitespace is in the allow list; so it isn't adding anything.

Reply:
Addressed